packages/account/src/Sections/Security/ClosingAccount/closing-account-pending-conditions/closing-account-pending-wrapper.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+
+type TClosingAccountPendingWrapperProps = {
+    title: JSX.Element;
+    description?: React.ReactNode;
+};
+
+const ClosingAccountPendingWrapper = ({
+    children,
+    title,
+    description,
+}: React.PropsWithChildren<TClosingAccountPendingWrapperProps>) => (
+    <div className='closing-account-error'>
+        <Text
+            as='div'
+            line_height='s'
+            size='xs'
+            weight='bold'
+            color='prominent'
+            className='closing-account-error__title'
+        >
+            {title}
+        </Text>
+        {description && (
+            <Text as='div' size='xxs' className='closing-account-error__description'>
```

Comment:
regarding both Texts here: `p` is better than `div` for `Text`, it's also a block element, and kind of makes more sense for text :)